lua/jq-playground/playground.lua

```diff
@@ -78,10 +78,16 @@ local function resolve_winsize(num, max)
   end
 end
 
-local function create_split_buf(opts)
+local function create_split_buf(opts, before_filetype_callback)
   local buf = vim.fn.bufnr(opts.name)
   if buf == -1 then
     buf = vim.api.nvim_create_buf(true, opts.scratch)
+
+    -- Run callback before setting filetype
```

Comment:
Why should it be done before setting the filetype if it's just for setting a buf var?